src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,14 @@
+$square-size: 100px;
+$calendar-padding: 10px;
+$calendar-gap: 1px;
+
+$days:
+  "mon" 0,
+  "tue" 1 * ($square-size + $calendar-gap),
+  "wed" 2 * ($square-size + $calendar-gap),
+  "thu" 3 * ($square-size + $calendar-gap),
+  "fri" 4 * ($square-size + $calendar-gap),
+  "sat" 5 * ($square-size + $calendar-gap),
+  "sun" 6 * ($square-size + $calendar-gap);
```

Comment:
you need to store only week day and the number
than go through this variable with cycle, and set correct margins to the needed selectors (formula should be written only once in the cycle, don't repeat it many times)